dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/internal/SSLComponent.java

```diff
@@ -332,6 +341,9 @@ private synchronized void processConfig(boolean updateSSLConfigManager) {
                                                              isServer,
                                                              transportSecurityEnabled,
                                                              repertoirePIDMap);
+                if (checkpointPhase != null) {
+                    SSLSocketFactory.getDefault();
```

Comment:
We need to check with the OpenJ9 team to make sure this is safe to do on the checkpoint side.  We need to ask @alon-sh.